docs/usage.md

```diff
@@ -6,28 +6,202 @@
+`python3 GA_csv_gen.py /path/to/gene_alignment_data/` This path should be the top level directory, e.g., the `treeval-resources/gene_alignment_data/` directory mentioned in the recommended directory structure above.
+
+This will print out quite a bit of information that looks like this:
+
+```bash
+============> CorvusMon1.bCorMon1 -- bird
+Generating CSV for:     CorvusMon1.bCorMon1
+Save Path:              /gene_alignment_data/bird/csv_data/CorvusMon1.bCorMon1-data.csv
+============> CorvusMoneduloides.bCorMon1 -- bird
+Generating CSV for:     CorvusMoneduloides.bCorMon1
+Save Path:              /gene_alignment_data/bird/csv_data/CorvusMoneduloides.bCorMon1-data.csv
+============> Gallus_gallus.UW_022020 -- bird
+Generating CSV for:     Gallus_gallus.UW_022020
+Save Path:              /gene_alignment_data/bird/csv_data/Gallus_gallus.UW_022020-data.csv
+============> Gallus_gallus.GRCg6a -- bird
+Generating CSV for:     Gallus_gallus.GRCg6a
+Save Path:              /gene_alignment_data/bird/csv_data/Gallus_gallus.GRCg6a-data.csv
+============> GallusGallus.GRCg7b -- bird
+Generating CSV for:     GallusGallus.GRCg7b
+Save Path:              /gene_alignment_data/bird/csv_data/GallusGallus.GRCg7b-data.csv
+```
+
+So what is happening is that it is walking the directory, identifying each unique {organism.Accession} and generating a csv summarising the data found in those directories into a csv with the following information, this looks like:
+
+```bash
+(base) dp24@/scripts$ head -n 5 /gene_alignment_data/bird/csv_data/Gallus_gallus.GRCg6a-data.csv
+org,type,data_file
+Gallus_gallus.GRCg6a,cds,/gene_alignment_data/bird/Gallus_gallus/Gallus_gallus.GRCg6a/cds/Gallus_gallus9002cds.MOD.fa
+Gallus_gallus.GRCg6a,cds,/gene_alignment_data/bird/Gallus_gallus/Gallus_gallus.GRCg6a/cds/Gallus_gallus28453cds.MOD.fa
+Gallus_gallus.GRCg6a,cds,/gene_alignment_data/bird/Gallus_gallus/Gallus_gallus.GRCg6a/cds/Gallus_gallus18005cds.MOD.fa
+Gallus_gallus.GRCg6a,cds,/gene_alignment_data/bird/Gallus_gallus/Gallus_gallus.GRCg6a/cds/Gallus_gallus6001cds.MOD.fa
+```
+
+This is all useful for the pipeline which generates job ids based on the org column, groups files by org and type columns and then pulls data from the data_file.
+
+#### HiC data Preparation
+
+Illumina HiC read files should be presented in an unmapped CRAM format, each must be accompanied by an index file (.crai) generated by samtools index. If your unmapped HiC reads are in FASTQ format, you should first convert them to CRAM format by using samtools import methods. Examples are below:
+
+##### Conversion of FASTQ to CRAM
+
+```bash
+samtools import -@8 -r ID:{prefix} -r CN:{hic-kit} -r PU:{prefix} -r SM:{sample_name} {prefix}_R1.fastq.gz {prefix}_R2.fastq.gz -o {prefix}.cram
+```
+
+##### Indexing of CRAM
+
+```bash
+samtools index {prefix}.cram
+```
+
+#### PacBio Data Preparation
+
+Find information on this here: [PacBio Data Prep](pacbio.md)
 
 ## Full samplesheet
 
```

Comment:
In this section, could you add links to the example yaml files that you use for the tests ? I think it'd be useful for people to actually see complete files.